src/main/docker/env.bash

```diff
@@ -16,6 +16,7 @@ export REMREM_PUBLISH_IMAGE="eiffelericsson/eiffel-remrem-publish:2.0.2"
 export JENKINS_IMAGE="bitnami/jenkins:2.138.3"
 export EI_BACKEND_IMAGE="eiffelericsson/eiffel-intelligence-backend:1.0.2"
 export EI_FRONTEND_IMAGE="eiffelericsson/eiffel-intelligence-frontend:1.0.3"
+export LDAP_IMAGE="osixia/openldap"
```

Comment:
Can you set a specific version, so we don't use latest image and risk of getting an new version of openldap that breaks our tests. E.g. "osixia/openldap:1.2.4"

Reply:
Done.